packages/protocol/lib/test-utils.ts

```diff
@@ -418,3 +423,110 @@ export async function addressMinedLatestBlock(address: string) {
   const block = await web3.eth.getBlock('latest')
   return isSameAddress(block.miner, address)
 }
+
```

Comment:
Nothing to be addressed in this PR, but @yorhodes @m-chrzan we should refactor this file, its huge